pkg/ebpf/c/bpf-common.h

```diff
@@ -0,0 +1,29 @@
+#ifndef BPF_COMMON_H
+#define BPF_COMMON_H
+
+#include <linux/bpf.h>
+#include <linux/tcp.h>
```

Comment:
You shouldn’t need to include `linux/tcp.h` as this file contains nothing related to TCP.